Resources/Prototypes/Nyanotrasen/Recipes/Reactions/drink.yml

```diff
@@ -3,9 +3,9 @@
   reactants:
     GrapeSoda:
       amount: 1
-#    Spaceacillin:
-#      amount: 1
-    Acetone:
+    Diphenhydramine:
```

Comment:
bruh this is way harder than cillin and acetone **nobody** will be able to make it

Reply:
yes, it shouldn't be *that* easy. This is the recipe we've been using since the rebase anyway. It was just reverted in the big merge.

Reply:
Also, Diphenhydramine makes way more sense, since it's the antihistamine.

Reply:
stellibinin is antihistamine too :trollface:

and le departmental interaction, and somewhat more possibility of ghetto lean

Reply:
diphen really isn't *that* hard to make, it's just that it's 2 steps to make the diethylamine first. There wasn't any complaints about it for the first 5 months it was this way.

Reply:
its not hard to make once but you have to keep making new beakers to make more which is annoying

Reply:
What do you mean more beakers?

Reply:
to reset the temperature since for whatever reason temperature will never go down on its own

Reply:
Oh, the beaker holds the temp....hmmm

Reply:
We need the opposite of a hotplate, a chiller.